changelog.txt

```diff
@@ -1,3 +1,17 @@
+4.24.0
+============
+
+New Features:
+**Performance improvement by reducing the number of calls to GetAllAccessTokens()**. See issue [#2204](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2204) for details.
+**MSAL .NET now sends the whether the token cache is serialized to telemetry**. See issue [#2185](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2185) for details.
+
+Bug Fixes:
+**MSAL .NET now do not force user to enter credentials for WAM AcquireTokenInteractive with picker**. See issue [#2233](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2233) for details. 
+**MSAL .NET now throws `RopcDoesNotSupportMsaAccounts` when ROPC is attempted with MSA account**. See issue [#2169](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2169) for details.
+**MSAL .NET now supports WithForceRefresh for AcquireTokenOnBehalfOfParameterBuilder**. See issue [#2232](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2232) for details.
+**Fix PlatformNotSupportedException in MacOS**. See issue [#2251](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2251) for details.
+**AcquireTokenOnBehalfOfAsync is now not visible in the Intellisense**. See issue [#2263](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2263) for details.
```

Comment:
```suggestion
```

Reply:
i don't think this is needed in the changelog

Reply:
or just cleaner IntelliSense? but yes. this is not needed